src/app/Scenes/MyCollection/MyCollection.tsx

```diff
@@ -74,7 +75,7 @@ const MyCollection: React.FC<{
 
   const toast = useToast()
 
-  const hasCollectedArtists = Number(me.myCollectionInfo?.artistsCount) > 0
+  const hasCollectedArtists = extractEdges(me.userInterestsConnection).length > 0
```

Comment:
What do you think about using `totalCount` instead? Actually, I think, both versions work fine 👍 

```suggestion
  const hasCollectedArtists = me.userInterestsConnection.totalCount)> 0
```

Reply:
good suggestion